src/Compilers/Test/Core/Assert/ConditionalFactAttribute.cs

```diff
@@ -230,6 +230,8 @@ public override bool ShouldSkip
 
     public class IsEnglishLocal : ExecutionCondition
     {
+        public static readonly IsEnglishLocal Instance = new();
```

Comment:
We don't use target-typed new in the compiler codebase, so that we can do textual search for `new Type`